Web/src/components/ui/toaster.tsx

```
@@ -0,0 +1,38 @@
+import { useToast } from "@/hooks/use-toast"
+import type { ToasterToast as ToastItem } from "@/hooks/use-toast"
+import {
+  Toast,
+  ToastClose,
+  ToastDescription,
+  ToastProvider,
+  ToastTitle,
+  ToastViewport,
+} from "@/components/ui/toast"
+
+export interface ToasterProps {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)